src/cephadm/cephadm

```diff
@@ -784,7 +784,7 @@ except NameError:
     TimeoutError = OSError
 
 
-class Timeout(TimeoutError):
+class Timeout(TimeoutError): # type: ignore
```

Comment:
which type error do you try to avoid here?

Reply:
variable possibly unbound. We know it isn't because of the above check, but linter is being annoying about it.

Reply:
I filled https://github.com/microsoft/pyright/issues/1363